widgetssdk/src/main/java/com/glia/widgets/call/CallView.java

```diff
@@ -201,6 +201,7 @@ public void startCall(String companyName,
                           String contextUrl,
                           boolean useOverlays,
                           Engagement.MediaType mediaType) {
+        Dependencies.getSdkConfigurationManager().setUseOverlay(useOverlays);
```

Comment:
`startCall` method is doing many different things here: setting configuration, initCall, onNavigatedToCall. Method could be broken down into multiple methods with less responsibilities.

Reply:
Is there any more appropriate layer (presenter, controller etc) to put that kind of code?

Reply:
Prefer not to do it atm. 
- breaking down is not related to this commit (will add some sort of technical debt task for that)
- its extra usecase for setting one variable

Atm. Views public interface requires variables

Will refac to using Configuration class ASAP (check PR notes)